src/Page.jsx

```diff
@@ -201,6 +201,7 @@ export default class Page extends Component {
         className={mergeClassNames('ReactPDF__Page', className)}
         ref={this.props.inputRef}
         style={{ position: 'relative' }}
+		{...{'data-page-number': page.pageIndex + 1}}
```

Comment:
This syntax is unnecessary and not optimized. Also, I believe it'd be better to use the functions we already have set up in this component, so this line should look like so:

`data-page-number={this.getPageNumber()}`

But I'm all for this idea! :)